packages/core/src/odata/v2/batch-request-builder.ts

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+import { errorWithCause } from '@sap-cloud-sdk/util';
+import {
+  Destination,
+  DestinationNameAndJwt,
+  DestinationOptions
+} from '../../scp-cf';
+import { BatchResponse } from '../common';
+import { parseBatchResponse } from '../common/request-builder/batch/batch-response-parser';
+import { BatchRequestBuilder } from '../common/request-builder/batch/batch-request-builder';
+import * as responseDataAccessor from './request-builder/response-data-accessor';
```

Comment:
I do not like this approach. I know the alternative of creating a instance of the `ResponseDataAccessor` interface for v2 and v4 is more tedious, but with this approach you do not need to have the implementing method to have the same name as the interface method.

Reply:
Also it is not the way we have done it in other places and I would like to have it consistent.

Reply:
Done